diff --git a/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h b/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
index bb59351..5046a81 100644
--- a/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
+++ b/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
@@ -10,6 +10,7 @@
 #include <array>
 #include <unordered_map>
 
+#include <folly/Format.h>
 #include <glog/logging.h>
 #include <react/debug/react_native_expect.h>
 #include <react/renderer/core/PropsParserContext.h>
@@ -77,7 +78,7 @@ inline folly::dynamic toDynamic(const YGValue& dimension) {
     case YGUnitPoint:
       return dimension.value;
     case YGUnitPercent:
-      return std::format("{}%", dimension.value);
+      return folly::sformat("{}%", dimension.value);
   }
 
   return nullptr;
